packages/esm-ward-app/src/ward-patient-card/row-elements/ward-patient-identifier.tsx

```diff
@@ -0,0 +1,43 @@
+import React from 'react';
+import { type WardPatientCardElement } from '../../types';
+import { type PatientCardElementConfig } from '../../config-schema';
+import { Tag } from '@carbon/react';
+
+const identifierCompareFunction = (patientIdentifier1, patientIdentifier2) => {
+  const sortByDate =
+    (!patientIdentifier1.preferred && !patientIdentifier2.preferred) ||
+    (patientIdentifier1.preferred && patientIdentifier2.preferred);
+  if (sortByDate) {
+    const patientIdentifier2Date = new Date(
+      patientIdentifier2.auditInfo.dateChanged || patientIdentifier2.auditInfo.dateCreated,
+    );
+    const patientIdentifier1Date = new Date(
+      patientIdentifier1.auditInfo.dateChanged || patientIdentifier1.auditInfo.dateCreated,
+    );
+    return patientIdentifier2Date.valueOf() - patientIdentifier1Date.valueOf();
+  }
+  if (patientIdentifier1.preferred) return -1;
+  if (patientIdentifier2.preferred) return 1;
+  return 0;
+};
+
+const wardPatientIdentifier = (config: PatientCardElementConfig) => {
+  const WardPatientIdentifier: WardPatientCardElement = ({ patient }) => {
+    const { identifierTypeUuid } = config;
+    const patientIdentifiers = patient.identifiers.filter(
+      (patientIdentifier) => patientIdentifier.identifierType?.uuid === identifierTypeUuid,
```

Comment:
Let's have identifierTypeUuid be optional as the concept of preferred identifier already exists in the backend.

```suggestion
      (patientIdentifier) => identifierTypeUuid == null || patientIdentifier.identifierType?.uuid === identifierTypeUuid,
```

Reply:
+1

Reply:
Made the change. Thanks @chibongho @mseaton .